### DISCUSSION TITLE
Restarting multihead replay fine-tuning for foundational model

### DISCUSSION THREAD

**Discussion by tdprice-858** [ACEsuit/mace#820] [General]
Hello all,

I was wondering if you could provide some guidance on best practices for restarting the multihead replay fine-tuning for the MPA-0 foundational model. Is it as simple as adding the --restart_latest tag and rerunning in the same directory?
I did not add any new data, and just wanted to run the fine-tuning for additional epochs to continue improving the accuracy.

I tried this and my model got worse, when previously the rmse on the forces was consistently decreasing. The checkpoint was correctly loaded and the dataset used was the same. 

Thank you for you time!

**Comment by ilyes319** [comment]
Can you please share your log file and checkpoint please, so we can debug?

**Reply by tdprice-858** [reply]
Thank you so much for the quick reply. I have attached my log file.
I did increase the force weight for the loss function during the rerun. But aside from that I believe everything else was the same.

**Comment by tdprice-858** [comment]
[fine-tuned_MACE-MPA-0-med_run-3.log](https://github.com/user-attachments/files/18832904/fine-tuned_MACE-MPA-0-med_run-3.log)

**Comment by GHLau** [comment]
Hi, how to set \--pt_train_file= \ when multihead replay fine-tune for the MPA-0 foundational model?
